spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/AppNode.java

```diff
@@ -32,6 +32,8 @@
 	private LabelNode label;
 
 	private ArgumentNode[] arguments;
+	
+	private boolean isLongLivedNonStreamApp = true;
```

Comment:
perhaps just 'nonManagedApp`?  since we create one of these using the `stream create` command, and is quite likely a Kafka app with multiple inputs/outputs, it feels off to say it is not a stream app....

Reply:
I was wrestling with the right name here. sounds good.